templates/layouts/html.html

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 <html>
 <head>
+  <base href="{% if NODE_ENV == "drupal" %}{{ npm_package_config_kstatic_drupalRemote}}{% else %}{{ npm_package_config_kstatic_base}}{% endif %}" />
```

Comment:
Sass just writes css, which takes base href into account (its a browser thing not a code thing).

There is a way to pass variables and functions at compile time to sass (https://github.com/sass/node-sass) but I doubt we need it.